domains/0y0s.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "0y0s"
```

Comment:
You should have additional contacts including an email field. Refer to docs for more information: https://docs.is-a.dev